### PR TITLE
BUG FIX: Fixed an issue whith uploaded files for Multisites

### PR DESCRIPTION
**Pull request by andrewlimaza**
BUG FIX: Fixed an issue of saved for preview and full URL's when using a multisite setup. This stores the uploads in the relevant site upload folder.

The URL's only supported single sites (non-Multisite setups).

### CODE REVIEW

File: classes/class.field.php [hunk 1]
```diff
@@ -316,17 +316,18 @@ function saveFile($user_id, $name, $value)
 				}
 			}
 
+			$upload_path = is_multisite() ? content_url('/uploads/sites/' . get_current_blog_id() . '/pmpro-register-helper/' . $user->user_login . '/') : content_url( '/uploads/pmpro-register-helper/' . $user->user_login . '/' );
```
Comment (sc0ttkclark) [inline classes/class.field.php:319]:
```suggestion
			// Our folder in the uploads directory that we want to save files to.
			$upload_dir  = '/pmpro-register-helper/' . $user->user_login . '/';

			// If multisite, prefix the directory with the current blog ID specific folder.
			if ( is_multisite() ) {
				$upload_dir = '/sites/' . get_current_blog_id() . $upload_dir;
			}

			// Get the full uploads directory URL we want to save files to.
			$upload_path = content_url( '/uploads' . $upload_dir );
```

**Comment by andrewlimaza** [comment]
Thanks @sc0ttkclark I committed your changes.

File: classes/class.field.php [hunk 2]
```diff
@@ -316,17 +316,27 @@ function saveFile($user_id, $name, $value)
 				}
 			}
 
+			// Our folder in the uploads directory that we want to save files to.
+			$upload_dir  = '/pmpro-register-helper/' . $user->user_login . '/';
```
Comment (sc0ttkclark) [inline classes/class.field.php:320]:
```suggestion
			$upload_dir = '/pmpro-register-helper/' . $user->user_login . '/';
```

File: classes/class.field.php [hunk 3]
```diff
@@ -316,17 +316,27 @@ function saveFile($user_id, $name, $value)
...
+
+			// If multisite, prefix the directory with the current blog ID specific folder.
+			if ( is_multisite() ) {
+				$upload_dir = '/sites/' . get_current_blog_id() . $upload_dir;
+			}
+
+			// Get the full uploads directory URL we want to save files to.
+			$upload_path = content_url( '/uploads' . $upload_dir );
```
Comment (sc0ttkclark) [inline classes/class.field.php:328]:
```suggestion
			$upload_path = content_url( '/uploads' . $upload_dir );

```

**Review by sc0ttkclark — APPROVED** [review]

**Pull request merged**